nft/nfts.yaml

```diff
@@ -188,7 +188,7 @@ paths:
                         type: string
                         description: 'String - Total number of NFTs in a given NFT collection.'
                       tokenType:
-                        $ref: '#/components/schemas/tokenType_response'   
+                        $ref: '#/components/schemas/tokenType_response'
```

Comment:
@alex-miao did u manually remove this or do u have some sort of linter that does this for you? would love to set it up lol

Reply:
Think it was a linter on VS Code, not certain which one but will send you the list of extensions I have.